src/components/BookFinder/BookList/BookList.tsx

```diff
@@ -26,7 +25,7 @@ export const BookList: FunctionComponent<BookListProps> = (props) => {
         }
       >
         <List.Item.Meta
-          title={item.volumeInfo.title}
+          title={<Link to={bookUrl}>{item.volumeInfo.title}</Link>}
           description={renderAuthors(item.volumeInfo.authors)}
```

Comment:
This is good for now, just note the following:
If the title becomes demanding or starts consisting of many HTML elements/react components, you should make a render function. Like you did in the prop below, the description prop. It's a minor code style detail, but a general rule is that the return/render of a component should be as clean as possible. That also applies to the render functions inside a component but in a less impactful manner (it is not as needed)

Reply:
ovo mi je super, skroz si u pravu! tenks!